README.md

```diff
@@ -29,12 +29,19 @@ It is possibile to snooze the notification issue for `x` time by closing it as *
 
 **Optional** A GitHub token.
 
-### `stale-days: 7`
+### **Deprecated** `stale-days: 7`
```

Comment:
since we are deprecating this, let's move it below the new option so it doesn't appear at the top anymore